### DISCUSSION TITLE
disable donwloading models

### DISCUSSION THREAD

**Discussion by yuriycherepnev** [danielgatis/rembg#684] [Q&A]
How to disable downloading models when docker container is up? i don't need models BiReFnet and others. I only use u2net.

**Comment by catscarlet** [comment]
Comment this line in Dockerfile
`RUN rembg d`

Build your own docker image

Download model by yourself and save it in somewhere. (in this example it's `/yourstorage2/libs/u2net/`).

Then run docker like this:
```
sudo docker run --gpus all -v /dev/dri:/dev/dri -v /yourstorage2/libs/u2net:/root/.u2net $PWD:/rembg rembg i -m birefnet-general input.png output.png
```
